Sources/Shared/Navigation/Navigator.swift

```diff
@@ -42,8 +42,9 @@ public final class Navigator {
     self.currentController = currentController
     Compass.routes = Array(navigationRouter().routes.keys)
 
+    // FIXME
     if let intentRouter = intentRouter?() {
-      Array(intentRouter.routes.keys)
+      let _ = Array(intentRouter.routes.keys)
```

Comment:
Oh, I forgot about this one, it should be `Compass.routes += Array(intentRouter.routes.keys)`